graphql/ui_plugin_build_baron.go

```diff
@@ -139,9 +143,19 @@ func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnIn
 
 	tickets, source, err = multiSource.Suggest(t)
 	if err != nil {
+		grip.Debug(message.Fields{
+			"message": "Chaya, graphql/ui_plugin_build_baron.go, 147",
+			"err":     err,
+			"tickets": tickets,
+			"source":  source,
+		})
 		return nil, projectNotFoundError, errors.New(fmt.Sprintf("Error searching for tickets: %s", err.Error()))
 	}
 	jql := t.GetJQL(bbProj.TicketSearchProjects)
+	grip.Debug(message.Fields{
```

Comment:
I fear logging every query (not only the errors) might be a lot.
If you agree, maybe line 154 could be moved before 146 and the jql could only be logged in the error case, together with the previous log message.